internal/app/timed/pkg/ntp/options.go

```diff
@@ -23,9 +23,9 @@ func defaultOptions() *NTP {
 	// defaults for minpoll + maxpoll
 	// http://www.ntp.org/ntpfaq/NTP-s-algo.htm#AEN2082
 	return &NTP{
-		Server:  "pool.ntp.org",
-		MaxPoll: MaxAllowablePoll * time.Second,
-		MinPoll: 64 * time.Second,
+		Server:  "ru.pool.ntp.org",
```

Comment:
temporary changes for debugging, will remove that

Reply:
I wonder if we can pick this default intelligently. Based on location of the machine.

Reply:
Not for this PR of course

Reply:
I read in the FAQ that `pool.ntp.org` is actually supposed to give out best server based on location, so probably we don't need much for defaults here.

Reply:
Oh nice.